libs/remix-lib/src/web3Provider/web3VmProvider.ts

```diff
@@ -227,9 +234,10 @@ export class Web3VmProvider {
 
   getCode (address, cb) {
     address = toChecksumAddress(address)
-    const account = toBuffer(address)
-    this.vm.stateManager.getContractCode(account, (error, result) => {
-      cb(error, hexConvert(result))
+    this.vm.stateManager.getContractCode(Address.fromString(address)).then((result) => {
+      cb(null, hexConvert(result))
```

Comment:
I see you have ignored `hexConvert ` above in code, would you like to do that here too?